demos/pom.xml

```diff
@@ -163,9 +163,6 @@
         <plugin>
```

Comment:
In parent, skip is set to true, shall we enforce the checkstyle check by default?